aslam_offline_calibration/ethz_apriltag2/CMakeLists.txt

```diff
@@ -1,41 +1,96 @@
-cmake_minimum_required(VERSION 3.0.2)
-project(ethz_apriltag2)
-
-add_definitions(-fPIC -O3)
+cmake_minimum_required(VERSION 3.28.0)
+project(ethz_apriltag2
+  VERSION 0.0.1
+  DESCRIPTION "ETHZ AprilTag2 library"
+  LANGUAGES CXX
+)
 
-find_package(catkin REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(OpenCV REQUIRED)
 
-catkin_package(
-  INCLUDE_DIRS include ${EIGEN3_INCLUDE_DIRS}
-  LIBRARIES ${PROJECT_NAME}
+set(CMAKE_BUILD_TYPE Release)
```

Comment:
@frneer nit^2: consider making Release a fallback only in case the user does not provide it.